reference/price_lists.v3.yml

```diff
@@ -1387,7 +1402,7 @@ paths:
 
         * Batch updates are supported by this endpoint, meaning that several price objects can be updated in one request. This allows you to do the same work as many individual requests to singleton endpoints
         * Batch requests support up to 1,000 items per request.
-        * Up to 2 concurrent batch upsert requests are supported with this API. Running more than the allowed concurrent requests in parallel on the **same store** will cause a 429 error and your additional requests will fail. You are encouraged to run requests sequentially with as many records per request as possible, in order to maximize performance.
```

Comment:
This one stays 2.  It's a batch endpoint for updating up to 1K items in one go and is a heavy lift.  Apologies if I wasn't clear about that earlier.